### PR TITLE
Change report form visibility

### PR DESCRIPTION
**Pull request by jiminbyun**
Previously, report form was visible when user loaded the page. Now, report form is only visible once user clicks on the report button.

### CODE REVIEW

File: web-app/src/main/webapp/script.js [hunk 1]
```diff
@@ -55,3 +55,11 @@ function handleLocationError(browserHasGeolocation, infoWindow, pos) {
   );
   infoWindow.open(map);
 }
+
+window.onload = function() {
+  document.getElementById("report-form").style.visibility = "hidden";
+};
+
+function showReportForm() {
+  document.getElementById("report-form").style.visibility = "visible";
```
Comment (AshleyLTW) [inline web-app/src/main/webapp/script.js]: We need to hide the home screen elements too! Also not sure if this is just outdated code, but screens are now denoted by two classes, home and report.

**Review by AshleyLTW — APPROVED** [review]

File: web-app/src/main/webapp/script.js (hunk 1 above)
Comment (jiminbyun) [reply web-app/src/main/webapp/script.js]: Will do in later PR on seperate branch with updated UI!

File: web-app/src/main/webapp/index.html [hunk 2]
```diff
@@ -84,7 +84,7 @@
     <div class="container u-full-width" id="dock">
     </div>
     <img id="report-button" src="images/report.svg" alt="button to report a new
-      incident">
+      incident" onclick="showReportForm()">
```
Comment (darrnshn) [inline web-app/src/main/webapp/index.html]: nit: I know we were doing this for the starter tasks, but I believe the Right Way (tm) is to add it via JavaScript:

```
document.addEventListener('load', () => {
   document.getElementById('report-button').addEventListener('click', showReportForm);
});
```

File: web-app/src/main/webapp/script.js (hunk 1 above)
Comment (darrnshn) [reply web-app/src/main/webapp/script.js]: note that `visibility` doesn't hide the space that the elements occupy. Setting `display: none` will completely hide the element and also the space that it occupies.

**Review by darrnshn — APPROVED** [review]

**Pull request merged**